spec/requests/users/registrations_spec.rb

```diff
@@ -9,17 +9,18 @@
       tags 'Authorization'
```

Comment:
I missed that before change that like so. There is no such thing as Authorization endpoints
```suggestion
      tags 'Authentication'
```